src/test/java/com/rocketden/main/service/RoomServiceTests.java

```diff
@@ -210,6 +212,38 @@ public void joinFullRoomFailure() {
     }
 
 
+    @Test
+    public void manyUsersJoiningAnInfinitelySizedRoomSuccess() {
+        /**
+         * Verify join room request fails when the room is already full
+         * Define five users, and add to the room
+         */
```

Comment:
Change this comment to reflect what this test case is actually doing - I like the addition of the comments by the way, that helps a lot, thank you!